test/images/agnhost/webhook/pods.go

```diff
@@ -77,6 +80,35 @@ func admitPods(ar v1.AdmissionReview) *v1.AdmissionResponse {
 }
 
 func mutatePods(ar v1.AdmissionReview) *v1.AdmissionResponse {
+	shouldPatchPod := func(pod *corev1.Pod) bool {
+		if pod.Name != "webhook-to-be-mutated" {
+			return false
+		}
+		return !hasContainer(pod.Spec.InitContainers, "webhook-added-init-container")
+	}
+	return applyPodPatch(ar, shouldPatchPod, podsInitContainerPatch)
+}
+
+func mutatePodsSidecar(ar v1.AdmissionReview) *v1.AdmissionResponse {
+	if sidecarImage == "" {
+		klog.Fatal("No image specified by the sidecar-image parameter")
```

Comment:
this will exit the webhook pod, but not surface back to the test... to return an error visible to the test, you would need to reject admission with an error (set Allowed=false, and add a status message)

Reply:
Done.